aip/general/0121.md

```diff
@@ -94,6 +94,33 @@ the standard methods. Custom methods offer the same design freedom as
 traditional RPC APIs, which can be used to implement common programming
 patterns, such as database transactions, import and export, or data analysis.
 
+### Consistency
+
+For methods that operate on a resource, the completion of those operations
+(either successful or with an error) **must** mean that the state of the
```

Comment:
Do we need to mention LRO here, or is that understood?

Reply:
I feel like it's understood, since "long-running operation" is a variation of "operation", after all.

But I see the value in clarifying that it's both.